manifests/pipecd/values.yaml

```diff
@@ -186,6 +186,31 @@ prometheus:
             - source_labels: [__meta_kubernetes_pod_container_port_name]
               action: keep
               regex: envoy-admin
+          metric_relabel_configs:
+            - source_labels: [__name__]
+              target_label: method
+              regex: envoy_cluster_grpc_(.+)_(.+)_(.+)_(.+)_upstream_rq_time_bucket$
+              replacement: $4
+            - source_labels: [__name__]
+              target_label: __name__
+              regex: envoy_cluster_grpc_(.+)_(.+)_(.+)_(.+)_upstream_rq_time_bucket$
+              replacement: pipecd_requests_duration_bucket
```

Comment:
Should we add `envoy` to the name? for example, `pipecd_envoy_requests_`

Reply:
for sure, or `pipecd_gateway_requests_` seems to be better

Reply:
because this is a relabel_config for the pipecd-gateway job

Reply:
Right. That is better. 👍